babel.config.json

```diff
@@ -4,8 +4,14 @@
       "bugfixes": true,
       "shippedProposals": true,
       "corejs": 3,
-      "useBuiltIns": "entry"
-    }]
+      "useBuiltIns": "entry",
+      "include": [
+        "@babel/proposal-nullish-coalescing-operator",
+        "@babel/proposal-optional-chaining"
```

Comment:
Fun fact: with our browserslist config, and our code, aside from these two transform plugins, `@babel/preset-env` does nothing.